packages/big-design/src/components/ProgressCircle/styled.tsx

```diff
@@ -11,12 +11,14 @@ export const StyledProgressCircle = styled.svg<ProgressCircleProps>`
   `}
 `;
 
-export const StyledCircle = styled.circle.attrs(({ size, theme }: any) => ({
-  // rem not usable for circle svg cx, cy, and r values in Safari 14
-  cx: theme.helpers.emCalc(getDimensions(size) / 2),
-  cy: theme.helpers.emCalc(getDimensions(size) / 2),
-  r: theme.helpers.emCalc(getDimensions(size) / 2 - getStrokeWidth(size) / 2),
-}))<ProgressCircleProps>`
+export const StyledCircle = styled.circle.attrs(
+  ({ size, theme }: { theme: DefaultTheme; size: ProgressCircleProps['size'] }) => ({
```

Comment:
I think we could just simplify this
```suggestion
export const StyledCircle = styled.circle.attrs<ProgressCircleProps>(({ size, theme }) => ({
```